app/models/course_user.rb

```diff
@@ -72,6 +72,7 @@ class CourseUser < ActiveRecord::Base
   scope :instructors, -> { staff }
   scope :students, -> { where(role: roles[:student]) }
 
+  scope :not_phantom, -> { where(phantom: false) }
```

Comment:
I gave up after 1 hour >_< There were a few suggestions (eg. visible, listed or public - credits to @kxmbrian ) but I felt that any positive noun deviated too much from 'phantom', so it doesn't seem intuitive enough. 

The other option is to rename the column from phantom or something else (with the converse indication, like public or visible), then this would make sense.